src/PaymentGateways/Gateways/PayPalStandard/PayPalStandard.php

```diff
@@ -240,4 +244,12 @@ public function getOptions()
          */
         return apply_filters('give_get_settings_paypal_standard', $setting);
     }
+
+    /**
+     * @unreleased
+     * @inerhitDoc
+     */
+    public function refundDonation(Donation $donation)
+    {
+    }
```

Comment:
@ravinderk What's the purpose of an empty method?

Reply:
@jason.adams I added a `refundDonation` method to `PaymentGatewayInterface`  class, so it is required to add this method to the gateway class.

Initially, I added an empty method to `PaymentGateway` class, but then I thought it would be good to add it to the gateway instead of the payment gateway abstract.

Reply:
@JasonTheAdams sure i will implement it

Reply:
resolved: I added `CanRefundDonation` interface